README.md

```diff
@@ -7,3 +7,67 @@ We're using the [OpenWeather API](https://openweathermap.org/current) to get wea
 You'll need to get your own API Key from their website (for free) and add it as an environment variable in a `.env` file. We have a template available as `example.env`.
 
 You'll be using React initially to build this. If you're new to React, check out the [website](https://reactjs.org) for some information on getting started! 
+
+# Project Setup for Windows OS:
+### 1. Fork the project:
+> _This creates a copy of the project into repositories of your GitHub account_
+
+### 2. Clone the forked repository:
+
+  You need to clone (download) it to your local machine using the below command in terminal
+```bash
+   $ git clone https://github.com/Your_username/prep-project-23.MAR.PREP.2.git
+```
+> Replace `Your_username` with your github username because your cloning the forked repository that exists in your github repositories.
+> _This creates a local copy of the repository in your local machine_
+
+### 3. Navigate to the forked repository:
+Once you have cloned the `prep-project-23.MAR.PREP.2` repository into your local machine, move into that folder using the change directory `cd` command on Linux/ Mac/ Windows
+```bash
+   $ cd prep-project-23.MAR.PREP.2
+```
+run the below command if you wish to open the project in Visual Studio code editor
+```
+  $ code .
+```
```

Comment:
Suggestion: Instead of giving command to open the folder in VSCode, simply write "Open the folder in your favourite editor/IDE" or something similar. This will ensure people don't get confused about why only this command.

Reply:
Sure I will change this. Thanks for the comment